src/main/java/org/entando/entando/aps/system/services/page/PageService.java

```diff
@@ -387,7 +388,7 @@ public PageDto movePage(String pageCode, PagePositionRequest pageRequest) {
         boolean moveUp = page.getPosition() > pageRequest.getPosition();
         try {
             if (page.getParentCode().equals(parent.getCode())) {
-                while (iterations-- > 0 && this.getPageManager().movePage(pageCode, moveUp));
+                while (iterations-- > 0 && this.getPageManager().movePage(pageCode, moveUp)) ;
```

Comment:
Didn't change anything

Reply:
I've applied the _reformat code_ and it added a space a the end of the line